lib/connection/connection.js

```diff
@@ -20,7 +20,7 @@ const { init: initEasyLogging } = require('../logger/easy_logging_starter');
 const GlobalConfig = require('../global_config');
 const JsonCredentialManager = require('../authentication/secure_storage/json_credential_manager');
 
-const PRIVATELINK_URL_SUFFIX = '.privatelink.snowflakecomputing.com';
+const REGEX_PRIVATELINK_SUFFIX = /.privatelink.snowflakecomputing.(com|cn)?$/;
```

Comment:
it's not correct when there will be a different top level domain - may we just leave /.privatelink.snowflakecomputing./?